### PR TITLE
Add ability to set license_template upon creation

### PR DESCRIPTION
**Pull request by Kyslik**
needs tests edited this in GitHub webGUI

### CODE REVIEW

File: lib/Github/Api/Repo.php [hunk 1]
```diff
@@ -196,19 +197,21 @@ public function create(
         $hasWiki = false,
         $hasDownloads = false,
         $teamId = null,
-        $autoInit = false
+        $autoInit = false,
+        $licenseTemplate = 'mit'
```
Comment (acrobat) [inline lib/Github/Api/Repo.php]: I'm not sure we should use MIT as the default value as github doesn't have a default value. So maybe just an empty string so we don't enforce incorrect default data

**Comment by Kyslik** [comment]
True that; I was trying to set it as an example; thanks.

Martin Kiesel

**Comment by m1guelpf** [comment]
@Kyslik You also have to fix the tests

**Comment by Kyslik** [comment]
@m1guelpf I know I have to, there is no time atm :( (I used github webgui to edit the file..., can not imagine to make changes to tests using webgui).

**Pull request closed without merging**